nucypher/cli/painting/staking.py

```diff
@@ -267,3 +275,49 @@ def paint_min_rate(emitter, staker):
     ~ Previously set ....... {prettify_eth_amount(raw_minimum)}
     ~ Effective ............ {prettify_eth_amount(minimum)}"""
     emitter.echo(rate_payload)
+
+
+def paint_staking_rewards(stakeholder, blockchain, emitter, past_periods, staking_address, staking_agent):
+    if not past_periods:
+        reward_amount = stakeholder.staker.calculate_staking_reward()
+        emitter.echo(message=TOKEN_REWARD_CURRENT.format(reward_amount=round(reward_amount, TOKEN_DECIMAL_PLACE)))
+        return
+
+    economics = stakeholder.staker.economics
+    seconds_per_period = economics.seconds_per_period
+    current_period = staking_agent.get_current_period()
+    from_period = current_period - past_periods
+    latest_block = blockchain.client.block_number
+    from_block = estimate_block_number_for_period(period=from_period,
+                                                  seconds_per_period=seconds_per_period,
+                                                  latest_block=latest_block)
+
+    argument_filters = {'staker': staking_address}
+    event_type = staking_agent.contract.events['Minted']
+    entries = event_type.getLogs(fromBlock=from_block,
+                                 toBlock='latest',
+                                 argument_filters=argument_filters)
+
+    rows = []
+    rewards_total = 0
```

Comment:
You can save yourself the hassle of having to later convert this value to NU (L322) by just tracking it as a NU object. Arithmetic operations can be performed on NU objects.

For example
```python
In [24]: n = NU(0, 'NU')

In [25]: n += NU(1000.123, 'NU')

In [26]: print(f"{n}")
1000.123 NU

In [27]: n += NU(2345.456, 'NU')

In [28]: print(f"{n}")
3345.579 NU
...
In [38]: nunit_value = NU(123456789123456789123, 'NuNit')

In [39]: str(nunit_value)
Out[39]: '123.456789123456789123 NU'

In [40]: n += nunit_value

In [41]: str(n)
Out[41]: '3469.035789123456789123 NU'
```